lib/xcodeproj/project/object/swift_package_remote_reference.rb

```diff
@@ -18,7 +18,7 @@ class XCRemoteSwiftPackageReference < AbstractObject
         #--------------------------------------#
 
         def ascii_plist_annotation
-          " #{isa} \"#{File.basename(display_name)}\" "
+          " #{isa} \"#{File.basename(display_name,".git")}\" "
```

Comment:
I think this might be the inverse here. If I make a fresh project in Xcode, add SPM dependencies and commit it, the `.git` is not there.

Then I open and save the project like `Xcodeproj::Project.open(project_path).save` and it (seemingly incorrectly) adds the `.git` suffix. This is covered in https://github.com/CocoaPods/Xcodeproj/issues/864